projects/S905/packages/initramfs/package.mk

```diff
@@ -0,0 +1,71 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2017-present Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="initramfs"
+PKG_VERSION=""
+PKG_REV="1"
+PKG_ARCH="any"
+PKG_LICENSE="GPL"
+PKG_SITE="https://libreelec.tv"
+PKG_URL=""
+PKG_DEPENDS_TARGET="toolchain libc:init busybox:init linux:init plymouth-lite:init util-linux:init e2fsprogs:init dosfstools:init"
+PKG_SECTION="virtual"
+PKG_SHORTDESC="initramfs: Metapackage for installing initramfs"
+PKG_LONGDESC="debug is a Metapackage for installing initramfs"
+
+PKG_IS_ADDON="no"
```

Comment:
drop `PKG_IS_ADDON="no"`

Reply:
I think this is a problem with github as this does not exist in my local copy

Reply:
Not sure what happened, as long it is shown here it is likely merged that way. Try to force push your current tree.

Reply:
It's happened before, force pushed a new version and its gone now :)